int32 id # scenario number
int32 type # 1: gazebo master, 2: high level controller
int32 n_path # number of paths
geometry_msgs/Point root
geometry_msgs/Point goal